<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}

background-color : white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>606</width>
      <height>379</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_divide">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pushButton_cos">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>COS</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_tan">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>TAN</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="pushButton_acos">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>ACOS</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="pushButton_ln">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>LN</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="pushButton_atan">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>ATAN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_sin">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>SIN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="pushButton_log">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>LOG</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5" colspan="3">
      <widget class="QPushButton" name="pushButton_AC">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_minus">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_left_bracket">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pushButton_right_bracket">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="pushButton_asin">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>ASIN</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_dot">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_mod">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>MOD</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="pushButton_sqrt">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>SQRT</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="8">
      <widget class="QLineEdit" name="result_show">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>61</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_multiple">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_x">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_0">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_pow">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_plus">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QPushButton" name="pushButton_equal">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>DEPOSIT</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5" colspan="3">
      <widget class="QPushButton" name="pushButtonCreditCalc">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CREDIT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Калькулятор</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
